### DISCUSSION TITLE
How to increase the speed of processing of blocks

### DISCUSSION THREAD

**Discussion by souvikhaldar** [btcsuite/btcd#2002] [Q&A]
How increase the speed of processing of blocks by providing more RAM and CPU to the btcd process? 
I don't see any option as such in the config, but btcd is using just 30% MB ram and 40% CPU for processing of blocks from genesis on RPI4 with 1 TB ssd.

**Comment by guggero** [comment]
The bottleneck is the UTXO set on disk, this will be vastly improved by https://github.com/btcsuite/btcd/pull/1955.

**Reply by souvikhaldar** [reply]
Looking forward to it.
